php-checks/src/main/java/org/sonar/php/checks/HardCodedSecretCheck.java

```diff
@@ -206,6 +222,13 @@ private Stream<Pattern> variableSecretPatterns() {
     return variablePatterns.stream();
   }
 
+  private Stream<Pattern> literalPatterns() {
+    if (literalPatterns == null) {
+      literalPatterns = toPatterns("=\\s*+(?<suffix>[^\\\\ &;#,|]+)");
```

Comment:
Currently, removing `\\s*+` and `;#,|` from the pattern doesn't make the tests fail. I guess we could have more tests to have better coverage on the regex.